uaclient/apt.py

```diff
@@ -256,3 +256,11 @@ def migrate_apt_sources(clean=False, cfg=None, platform_info=None):
                 'Disabled %s after package upgrade/downgrade. %s',
                 entitlement.title, details)
         entitlement.enable()  # Re-enable on current series
+
+
+def is_pkg_installed(pkg_name: str) -> bool:
```

Comment:
given that there are ~11 packages  that coule be installed, do we want to batch this with a single dpkg -l and parsing instead for shelling out for each package?

Reply:
Good question; I'll do some profiling locally. Would you be happy with us changing that in a separate PR?

Reply:
Yes that works fine techdebt etc. I was wondering about something like get_installed_packages(pkg_names=())

Reply:
Given that this will happen as a follow-up, is this approved, or are you still working through your review?